src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
+
+VERSION_MAJOR = 1
+VERSION_MINOR = 3
+VERSION_REVISION = 0
+VERSION_NUMBER = (VERSION_MAJOR * 1000000 + VERSION_MINOR * 1000 + VERSION_REVISION)
```

Comment:
I agree that they should probably be removed, since, if they are required, then they should be obtained from `__version__`.

However, defining them in the `client` module certainly feels like the wrong place, given it has sibling modules.

I thought that develop was now permitting minor breaking changes anyway.